app/views/patients/successful_schedule.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="row text-center">
```

Comment:
Not a new file, renamed from `successfull` to `successful` and updated the file a bit.